src/config.rs

```diff
@@ -101,6 +101,11 @@ pub struct ClientConfig {
     pub retry_base_ms: Option<i64>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub retry_ceiling_ms: Option<i64>,
+    //Note this field does not belong in the hypersync_client::ClientConfig
+    //however the convert function will still work so long as deny_unknown_fields
+    //is not implemented in the deserializer
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub enable_checksum_addresses: Option<bool>,
```

Comment:
I've added this config item just to the node js client. Since this is what is responsible for turning the address bytes into a string.

See the note that it should still serialize fine to the hs client config with the extra field.